stockpile/roles/postgresql/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+  # Collect PostgreSQL related configuration
+  # Requires the postgresql.conf to be present
+  # under the /var/lib/pgsql/data directory
+  # TODO: Make this more general to read from
+  #       any location where the configuration
+  #       lives.
```

Comment:
Could we pass the directory via a var?

Reply:
@jtaleric We can pass this manually maybe through a var or we can also auto-detect it by running something like this:
find / -type f -name "postgresql.conf" | head -1
and then setting a variable using set_fact

What do you think?